src/WebJobs.Script/Description/Proxies/ProxyClientExecutor.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.AppService.Proxy.Client.Contract;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal static class ProxyClientExecutor
```

Comment:
How about an extension method? 

---
In reply to: [133530902](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r133530902) [](ancestors = 133530902)